lldb/examples/python/templates/scripted_platform.py

```diff
@@ -10,9 +10,6 @@ class ScriptedPlatform(metaclass=ABCMeta):
 
     Most of the base class methods are `@abstractmethod` that need to be
     overwritten by the inheriting class.
-
-    DISCLAIMER: THIS INTERFACE IS STILL UNDER DEVELOPMENT AND NOT STABLE.
-                THE METHODS EXPOSED MIGHT CHANGE IN THE FUTURE.
```

Comment:
Why remove the disclaimer? Is this no longer true?

Reply:
I'd expect at this point the extension to be stable.